test/unit/vdom/elements.js

```diff
@@ -0,0 +1,34 @@
+import element from '../../lib/element';
+import { symbols, vdom } from '../../../src/index';
+
+const shadowDomV0 = !!document.createElement('div').createShadowRoot;
```

Comment:
What would happen here in Chrome if we want to use named slots with skate?
It would be `true`, since it's native property in Chrome, but everything would be polyfilled by namedslots because we are using expression `if (!('attachShadow' in document.createElement('div'))) {`and there is no native `attachShadow` in there. But we are using `slot` insead of `content`, so everything would be broken.